esprima.js

```diff
@@ -6173,7 +6186,11 @@ parseYieldExpression: true, parseAwaitExpression: true
 
     function parseClassImplements() {
         var id, implemented = [], marker, typeParameters;
-        expectContextualKeyword('implements');
+        if (strict) {
+            expectKeyword('implements');
```

Comment:
Oh, and `static` is a reserved word in ES5 -- but that was removed in ES6 when it was realized that it didn't need to be for class method positions